addons/common/functions/fnc_checkPBOs.sqf

```diff
@@ -10,24 +10,23 @@
  *   1 = Warn permanently
  *   2 = Kick
  * 1: Check all PBOs? <BOOL> (default: false)
- * 2: Whitelist <STRING> (default: "")
+ * 2: Whitelist as list of addons. Regex patterns can be used to match addons. Addon name will be matched case-insensitive by default. <STRING> (default: "")
  *
  * Return Value:
  * None
  *
  * Example:
- * [0, false, ""] call ace_common_fnc_checkPBOs
+ * [0, true, "acex_headless,cba_.*"] call ace_common_fnc_checkPBOs
  *
  * Public: Yes
  */
 
 params ["_mode", ["_checkAll", false], ["_whitelist", "", [""]]];
 TRACE_3("params",_mode,_checkAll,_whitelist);
 
-// Lowercase and convert whiteList string into array of strings
-_whitelist = toLowerANSI _whitelist;
```

Comment:
Can you elaborate why you removed this?

Reply:
We perform case-insensitive string checks, so there is no need to convert to lowercase.